cluster/ci/config/prow/plugins.yaml

```diff
@@ -1155,6 +1155,10 @@ config_updater:
       name: prow-job-cluster-launch-installer-openstack-e2e
       additional_namespaces:
       - ci-stg
+    ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml:
```

Comment:
Can you propose the changes to this file in a separate PR to merge first?

Reply:
Moved to this new PR #4378